### DISCUSSION TITLE
Instant Metting 화면 오류 제보

### DISCUSSION THREAD

**Discussion by act-lamos** [metamong-friends/metamong-friends.github.io#3] [제보하기]
![image](https://github.com/metamong-friends/metamong-friends.github.io/assets/29725145/b04fe0d6-bf82-4695-934b-850f559a5677)

재현 순서

1. instant Metting 생성
2. private로 전환
3. 이동하여 instant Metting 종료
4. 다시 접근하여 instant Metting 생성시 public이지만 Private에 볼드처리 되어있습니다

**Comment by shlee303** [comment]
토글 관련 이슈 확인하겠습니다! 제보 감사합니다~!!

**Comment by shlee303** [comment]
처리 완료 됐습니다!
